docs/README.md

```diff
@@ -0,0 +1,15 @@
+# Assumptions
```

Comment:
Maybe we might want to add a comment about creating an AKV instance here just to  make it a bit easier to understand.

At a glance I was not sure `scripts/create_member_cert_akv.sh` creates an AKV instance or I need to make one. You can figure it out just by trying, but it would nice if it's clarified

Reply:
done